tests/common/util/MockLoadBalancer.hpp

```diff
@@ -48,3 +53,36 @@ struct MockLoadBalancer {
         (const)
     );
 };
+
+struct MockNgLoadBalancer : etlng::LoadBalancerInterface {
```

Comment:
Maybe rename this to `MockNgLoadBalancerImpl` and create an alias:
```cpp
using MockNgLoadBalancer = std::shared_ptr<testing::NiceMock<MockNgLoadNalancerdimpl>>;
```

Reply:
Let's do this in a separate PR so that we can make a util and fix other places as well 👍

Reply:
In #1813